ui/templates/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /dashboard/
+Disallow: /learner/
+Disallow: /learners/
+Disallow: /api/
```

Comment:
Can you review urls.py for other routes? I'm thinking `/cms/`, `/profile/` and `/financial_aid/` should also be blocked

Is `/admin/` typically blocked?